IDE/MonoDevelop/MonoDevelop.MonoGame.iOS/templates/Shared/Program.cs

```diff
@@ -0,0 +1,83 @@
+#region Using Statements
+using System;
+using System.Collections.Generic;
+using System.Linq;
+#if MONOMAC
+using MonoMac.AppKit;
+using MonoMac.Foundation;
+#elif __IOS__
+using Foundation;
+using UIKit;
+#endif
+using ${ProjectName};
+#endregion
+
+
+namespace ${ProjectName}.iOS
```

Comment:
The platform addition to the namespace again.

Same with the project name might not be a valid namespace.